docker-compose.static.yml

```diff
@@ -0,0 +1,41 @@
+# Container does not perform any initialization aside from importing env-vars from `confd`.
```

Comment:
So this implies that any time `docker-compose.local.yml` is updated, this file needs to be updated and kept in sync?

Reply:
Yes